tests/cephfs/nfs-ganesha_basics.py

```diff
@@ -49,8 +49,8 @@ def run(ceph_cluster, **kw):
             # Create cephfs nfs export
             nfs_client[0].exec_command(
                 sudo=True,
-                cmd=f"ceph nfs export create cephfs {fs_name} {nfs_name} "
-                f"{nfs_export_name} path={path}",
+                cmd=f"ceph nfs export create cephfs {nfs_name} "
```

Comment:
This is specific to 5.1 only or does it work for 5.0 also

Reply:
Yes, It's applicable only on 5.1, I am working on the same thing.. Will update in some time